packages/crypto/Cargo.toml

```diff
@@ -16,7 +16,7 @@ ark-bls12-381 = "0.4.0"
 ark-ec = { version = "0.4.2", features = ["parallel"] }
 ark-ff = { version = "0.4.2", features = ["asm", "parallel"] }
 ark-serialize = "0.4.2"
-cosmwasm-core = { version = "2.1.3", path = "../core" }
+cosmwasm-core = { version = "2.2.0-rc.1", path = "../core" }
```

Comment:
nit: We can consider using `[workspace.dependencies]` section in the main Cargo.toml file for CW related crates